pkg/cluster/meta.go

```diff
@@ -27,6 +29,12 @@ func WithOwnerReference(ownerReferences ...metav1.OwnerReference) MetaOptions {
 	}
 }
 
+func OwnedBy(owner metav1.Object, scheme *runtime.Scheme) MetaOptions {
+	return func(obj metav1.Object) error {
+		return controllerutil.SetOwnerReference(owner, obj, scheme)
+	}
+}
```

Comment:
I sneaked in this little helper, not used here, but I am cooking some stuff where I want to set:

```
return cluster.CreateOrUpdateConfigMap(
		r.Client,
		&corev1.ConfigMap{
			ObjectMeta: metav1.ObjectMeta{
				Name:      "platform-capabilities",
				Namespace: r.DataScienceCluster.DSCISpec.ApplicationsNamespace,
			},
			Data: componentCapabilitiesJson,
		},
		cluster.OwnedBy(dscInstance, r.Scheme),
		cluster.WithLabels(
			labels.K8SCommon.PartOf, "opendatahub",
			labels.K8SCommon.ManagedBy, "opendatahub-operator", 
		),
	)
```

Reply:
Can remove, that was actually accidental.